packages/fx-core/src/plugins/resource/bot/v2/plugin.ts

```diff
@@ -303,23 +283,65 @@ export class TeamsBotV2Impl {
     throw new Error("Invalid programming language");
   }
 
-  private resolveScenarios(
-    actRoles: PluginActRoles[],
-    triggers: BotTrigger[],
-    hostType: HostType
-  ): string[] {
-    const scenarios: string[] = [];
-    if (hostType === HostType.Function) {
-      if (actRoles.includes(PluginActRoles.Notification)) {
-        scenarios.push(TemplateProjectsScenarios.NOTIFICATION_FUNCTION_BASE_SCENARIO_NAME);
-        triggers.map((trigger) => scenarios.push(TriggerTemplateScenarioMappings[trigger]));
+  private resolveScenariosForMessageExtension(
```

Comment:
I still don't understand why we need to have two method to resolve scenario for ME and Bot, let's sync this offline.

Reply:
These methods are removed.